src/main/java/de/zalando/aruha/nakadi/service/subscription/model/Session.java

```diff
@@ -0,0 +1,42 @@
+package de.zalando.aruha.nakadi.service.subscription.model;
+
+import java.util.UUID;
+
+public class Session {
+    public final String id;
+    public final int weight;
```

Comment:
It's the estimation of client's ability to take load. For the first implementation it's always 1. But it could change in time.